### PR TITLE
Add Telemetry for GitHub Action

### PR DESCRIPTION
**Pull request by vincent1173**

### CODE REVIEW

File: aks/Dockerfile [hunk 1]
```diff
@@ -8,6 +8,8 @@ LABEL com.github.actions.description="Creates a service and a deployment with th
 LABEL com.github.actions.icon="triange"
 LABEL com.github.actions.color="blue"
 
+ENV GITHUB_ACTION_NAME="Deploy a container image to an AKS cluster"
+
```
Comment (ds-ms) [inline aks/Dockerfile]: Check if there is any max length for the `AZURE_HTTP_USER_AGENT` header if there is, does this fit in the length?

**Review by thesattiraju — APPROVED** [review]

File: aks/Dockerfile (hunk 1 above)
Comment (vincentdass) [reply aks/Dockerfile]: agreed, we could have a key like "AKS_ACTION".
Will check with PM

File: aks/entrypoint.sh [hunk 2]
```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 set -e
-export AZURE_HTTP_USER_AGENT="GITHUBACTIONS_${GITHUB_ACTION}_${GITHUB_REPOSITORY}"
+export AZURE_HTTP_USER_AGENT="GITHUBACTIONS_${GITHUB_ACTION_NAME}_${GITHUB_REPOSITORY}"
```
Comment (dikhakha) [inline aks/entrypoint.sh]: what's the benefit of using action name as is v/s descriptive name ? Just for my understanding, why are we making this change?

Comment (vincentdass) [reply aks/entrypoint.sh]: we were passing GitHub action which was "custom name provided by user", but we need to pass actual GitHub action (AZ CLI / AZ Web App deploy) for telemetry

**Review by SumiranAgg — APPROVED** [review]

**Review by dikhakha — APPROVED** [review]

**Review by kmkumaran — APPROVED** [review]

**Pull request merged**